packages/containerd/containerd-config-toml_k8s_containerd_sock

```diff
@@ -36,16 +36,12 @@ enable_unprivileged_ports = {{settings.container-runtime.enable-unprivileged-por
 enable_unprivileged_icmp = {{settings.container-runtime.enable-unprivileged-icmp}}
 {{/if}}
 
-[plugins."io.containerd.grpc.v1.cri".containerd]
-default_runtime_name = "shimpei"
-
-[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.shimpei]
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
```

Comment:
Do we still need to specify `default_runtime_name` somewhere, or does it default to `runc`?